EIPS/eip-5792.md

```diff
@@ -263,7 +271,6 @@ For each chain on which a wallet can submit multiple calls atomically, the walle
   },
   "0x14A34": {
     "atomicBatch": {
-      "supported": true
```

Comment:
accident? we want this field here

Reply:
ah i see. this is a little confusing to me, i think we'd actually expect to just not see the `atomicBatch` field at all. thats my understanding of what you've written above too. can we just leave this `true` example?

Reply:
ok